flake.nix

```diff
@@ -9,15 +9,15 @@
       let
         pkgs = nixpkgs.legacyPackages.${system};
       in
-      {
+      rec {
         packages = rec {
           binsider = pkgs.rustPlatform.buildRustPackage {
             name = "binsider";
             src = ./.;
             cargoLock = {
               lockFile = ./Cargo.lock;
             };
-            checkType = "debug";
+            buildType = "debug";
```

Comment:
Is there some specific reason why you changed the build type to debug?
I would think that most users would want a fast release build

Reply:
i agree about most users would want release but i don't see a way to build in debug mode on ci without this change in a flake

Reply:
i would be really happy to change it to release for building if someone has idea how to make it build in debug only on CI

Reply:
And why would you want to build in debug mode in CI? Just to build it faster? 
You could also just create two different packages, one that builds in debug and one in release.
And then set the default to the release one and the check to the debug one

Reply:
Yes, that was the reason. Thx for the tip